duckduckgo/iOS
# Autofill Batch 2 in browser experience changes

Task/Issue URL: https://app.asana.com/0/0/1202854530787111/f
Tech Design URL:
CC:

**Description**:

- Prevent autofill being accidentally disabled
  - Copy updates
  - Alert box to confirm disable 
  - Note that the new logic around applying new rules for when to show the confirm disable alert is not part of this PR 
- Make sure people know that DDG is saving passwords updates



**Steps to test this PR**:
1. Confirm that copy on first save sheet is correct
2. Confirm first save sheet is shown for all normal save sheets
3. Test that alert prompts to confirm disabling autofill after 3 rejections and that buttons perform as expected

Note that the new logic around applying new rules for when to show the confirm disable alert is not part of this PR 



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
